test/phabricator_test/test_phabricator.py

```diff
@@ -0,0 +1,356 @@
+from mock import patch
+import yaml
+import unittest
+from os.path import join, dirname
+try:
+    # Python 3.3 >
+    from unittest.mock import MagicMock
+except:
+    from mock import MagicMock
+from unittest import TestCase
+from reviewrot.phabricatorstack import PhabricatorService, PhabricatorReview
+from reviewrot.basereview import BaseService, BaseReview, LastComment
+import datetime
+from mock_phabricator import MockPhabricator
```

Comment:
Non blocking, just a nitpick, you can close this comment without any action.

The order of imports should usually be divided into three groups. First group are imports using build-in modules, second group should consists imports using 3rd party modules, and last one should contains project imports. That being said, the imports should be something like:

```python
import datetime
import unittest
from unittest.mock import MagicMock
from os.path import join, dirname

import yaml

from reviewrot.phabricatorstack import PhabricatorService, PhabricatorReview
from reviewrot.basereview import BaseService, BaseReview, LastComment
from mock_phabricator import MockPhabricator
```